### DISCUSSION TITLE
Allowing other accounts on different devices to download from their own watchlist

### DISCUSSION THREAD

**Discussion by Xenon1** [itsToggle/plex_debrid#120] [Q&A] [answered]
Tried giving my dad access to the plex media server which works. He can stream whatever is already on there but if he adds something to his own playlist, the script will not scrape. He has been added as a user for both the web app and his smart TV, see here:
[redacted]
Is there any other step i need to complete?

**Comment by Xenon1** [comment]
Update: removed the user at line 17 since i realised it was the access token for my personal web browser

**Comment by Xenon1** [comment]
okay little update in my attempt to de bug anything my side. it is reading watchlist content from my dad, and removes from all watchlists when the media is added. However if my dad adds something to his watchlist that isnt in mine, the script wont scrape for it. see image:
![image](https://user-images.githubusercontent.com/28608526/190274231-e79b24b1-c38e-45ad-99ec-014c1bb8d09b.png)

**Comment by itsToggle** [comment, marked as answer]
Hey, Plex tokens should remain private. I've removed the sensitive data from your post.

a simple question at first: is he a managed user or does he have his own Plex account? Only users with a Plex account have access to the discover feature and the Plex watchlist. From what you are describing I'm assuming he has a Plex account and access to the discover feature, but I wanted to make sure.

It sounds a bit like all user tokens youve added are linked to your account, and not to yours and Gary's. Plex_debrid will only print "removed from user X's watchlist", if the item really is in that users watchlist. Was this the case for "Free Guy"? Was that movie in both Gary's and your watchlist? Make sure that the token you add really is linked to Gary's Plex account.

On a side note: you only need to add one token per user, not per device. The Plex watchlist will sync across all devices :)

**Reply by Xenon1** [reply]
Ahh i didn't know that about plex tokens, so thanks for doing that. The issue was the plex token like you said. When i logged into his account to get his token, i logged in through app.plex.tv and not normal plex.tv. This then didn't refresh https://plex.tv/devices.xml to get his token. Thanks for the quick solution works perfectly now

**Reply by itsToggle** [reply]
I should make that clearer in the readme! Awesome that it's working now :)
